R/df_to_exb.r

```diff
@@ -0,0 +1,340 @@
+#' Dataframe to .exb
+#'
+#' Convert a dataframe to Exmaralda .exb
+#'
+#' @author Valeria Buntiakova <valleriabun@gmail.com>
+#'
+#' @param df an R dataframe object that contains columns named 'tier', 'tier_name', 'content', 'time_start', 'time_end' and 'id'
+#' @param name transcription name
+#' @param outputPath filepath
```

Comment:
I'd like to unify arguments' names across the package. So it's better to split `outputPath` to `output_file` and `output_dir` see `create_viewer()` function for an example.